src/eslintDefaultOptions.ts

```diff
@@ -1,134 +1,134 @@
-  "plugin:json/recommended",
-  "plugin:lit/recommended",
-  "plugin:lodash/recommended",
-  "plugin:monorepo/recommended",
-  //"plugin:@mysticatea/es2015",
-  //"plugin:@mysticatea/+eslint-plugin",
-  "plugin:perfectionist/recommended-natural",
-  "plugin:prettier/recommended",
-  "plugin:react/recommended",
-  "plugin:security/recommended",
-  "plugin:tailwindcss/recommended",
-  "plugin:test-selectors/recommended",
-  "plugin:you-dont-need-lodash-underscore/compatible"
-]
-
-const typescriptConfigs: string[] = [
-  "plugin:node/recommended",
-  "plugin:@angular-eslint/recommended",
-  "plugin:@angular-eslint/template/process-inline-templates",
-  "plugin:@typescript-eslint/eslint-recommended",
-  "plugin:@typescript-eslint/recommended",
-  "plugin:node/recommended"
-]
-
-export const defaultOptions: ESLint.Options = {
-  baseConfig: {
-    extends: baseConfigs,
-    env: {
-      es6: true,
-      node: true,
-      browser: true,
-      commonjs: true,
-      jquery: true,
-      phantomjs: true,
-      jasmine: true,
-      mocha: true,
-      amd: true,
-      worker: true,
-      qunit: true,
-    },
-    ignorePatterns: ["node_modules/*", "dist/*", "bin/*", "build/*", "docs/tests/*"],
-    plugins: pluginsNames,
-    parser: "@typescript-eslint/parser",
-    parserOptions: {
-      ecmaFeatures: {
-        "jsx": true,
-      },
-      requireConfigFile: false,
-      sourceType: "module",
-    },
-    root: true,
-    settings: {
-      "node": {
-        "tryExtensions": [".js", ".json", ".node"],
-      },
-      "import/parsers": {
-        "@typescript-eslint/parser": [".ts", ".tsx"],
-      },
-      "import/resolver": {
-        typescript: {
-          alwaysTryTypes: true,
-        },
-      },
-      jest: {
-        version: 26,
-      },
-      react: {
-        version: "18.2.0",
-      },
-    },
-    overrides: [
-      {
-        files: ["**/*.ts", "**/*.tsx"],
-        extends: typescriptConfigs,
-        parserOptions: {
-          project: ['./tsconfig.json'],
-          sourceType: "module",
-        },
-      },
-      {
-        files: ["**/*.vue"],
-        parser: require.resolve("vue-eslint-parser"),
-        parserOptions: {
-          ecmaVersion: 2020,
-          sourceType: "module",
-        },
-      },
-      {
-        files: ["**/*.jsx"],
-        parser: "@babel/eslint-parser",
-        parserOptions: {
-          babelOptions: {
-            presets: ["@babel/preset-react"],
-          },
-        },
-      },
-      {
-        files: ["**/*.html"],
-        extends: [
-          "plugin:@angular-eslint/template/recommended",
-          "plugin:@angular-eslint/template/accessibility"
-        ]
-      },   
-      {
-        files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
-        extends: ['plugin:testing-library/react'],
-      },
-    ],
-  },
-}
-
-export const defaultEngine = new ESLint(defaultOptions)
+import { ESLint } from "eslint"

```

Comment:
Why did you change this file? Shouldn't it be in a separate PR?
This changes the settings for all Codacy users.